.github/workflows/ci_s2ntests.yml

```diff
@@ -0,0 +1,377 @@
+---
+name: s2nCodeBuildTesting
+
+on:
+  pull_request:
+    branches:
+      - master
+jobs:
+  integ:
+    needs: unit
+    runs-on: ubuntu-18.04
+    strategy:
+      matrix:
+        openssl_version:
+          - openssl-1.0.2
+          - openssl-1.0.2-fips
+          - openssl-1.1.1
+          - libressl
+        gcc_version:
+          - 6
+          - 9
+      fail-fast: true
+    steps:
+      - uses: actions/setup-node@v1
+      - name: Configure AWS Credentials
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          aws-region: us-west-2
+      - uses: hmarr/debug-action@master
```

Comment:
Did you mean to leave this in there?

Reply:
No, the debug should come out.  I originally had ~6 workflows and the jobs were being killed with `rate exceeded` errors.  Without explicit documentation on launch rate, we could setup a load test to find the limits.